src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -389,6 +389,13 @@ export default {
 	}
 }
 
+:deep(.text-editor__wrapper .paragraph-content) {
+	margin-bottom: 0!important;
+}
+:deep(.text-editor__wrapper div.ProseMirror > *:first-child) {
+	margin-top: 0!important;
+}
```

Comment:
Not sure I fully get those two rules try to achieve, shouldn't we just target something like:

```
:deep(.text-editor__wrapper .paragraph-content:last-child)
```

Reply:
thank you! Took your suggestion!

Regarding:

```
:deep(.text-editor__wrapper div.ProseMirror > *:first-child) {
	margin-top: 0!important;
}
```

still think this margin-top have to be removed too
![Screenshot from 2024-07-23 14-56-35](https://github.com/user-attachments/assets/d1e5a78a-e682-4c95-aeb4-e816da3e2198)

Reply:
Ah right, that makes sense to drop then as well of course 👍